### DISCUSSION TITLE
How to achieve browsing and selecting folder paths

### DISCUSSION THREAD

**Discussion by xiaozhiagi** [zauberzeug/nicegui#1460] [Q&A]
### Question

How to achieve browsing and selecting a folder path? For example, I need to click "Browse" on the front end, and a folder selection window will pop up. Click "Open" to select the absolute path of the folder

**Comment by falkoschindler** [comment]
Please have a look into https://github.com/zauberzeug/nicegui/discussions/283 and https://github.com/zauberzeug/nicegui/tree/main/examples/local_file_picker

**Reply by usuall** [reply]
Thanks. 
Can you add a modified date in addition to the file name (folder name)? And based on the file name, the revision date, please improve it so that only the ascending, descending, and specific extensions can be seen.
